Dockerfile

```diff
@@ -114,8 +114,7 @@ RUN apk add --no-cache \
         icu \
         libldap \
         libpng \
-        libzip \
-        openldap && \
```

Comment:
so if im right, we only need the libldap?
in the php ext base build for alpine is still an openldap - also not needed?

Reply:
I removed that too, you were right.

Reply:
forgotten to check-in this changes?